server/chainquery/models/ClaimModel.js

```diff
@@ -145,19 +150,19 @@ export default (sequelize, {
       set() { },
     },
     is_filtered: {
-      type: INTEGER,
+      type: BOOLEAN,
```

Comment:
This is an old column. I need to delete this. Not sure why it is in the schema. I will create an issue for it.